openmc/deplete/abc.py

```diff
@@ -704,6 +734,59 @@ def __init__(self, operator, timesteps, power=None, power_density=None,
         self.timesteps = asarray(seconds)
         self.power = asarray(power)
 
+        if isinstance(solver, str):
+            # Delay importing of cram module, which requires this file
+            if solver == "cram48":
+                from .cram import CRAM48
```

Comment:
Is there a reason these imports are not done up top?

Reply:
Yeah, we get a circular import otherwise. `cram` imports `abc.DepSystemSolver` to make the `IPFCramSolver` and `CRAMXX` functions